core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -361,4 +361,6 @@ private TableProperties() {}
   public static final boolean UPSERT_ENABLED_DEFAULT = false;
 
   public static final String HMS_TABLE_OWNER = "hms_table_owner";
+  public static final String HMS_DB_OWNER = "hms_db_owner";
```

Comment:
I feel like we probably want to update these properties to be consistent with the rest of the table properties convention (i.e. `hive.metastore.table.owner`, `hive.metastore.database.owner`, `hive.metastore.database.owner-type`).  Is there a specific reason for the proposed property names?

Reply:
Also, @gaborkaszab is `user` the only owner type for tables?

Reply:
@danielcweeks yes, currently the table owner is for "USER" types.
About the naming, when I introduced HMS_TABLE_OWNER frankly I wasn't aware of the existing naming convention, so I'm absolutely open to rename them to follow how other properties are named.